rbac/server/api/users.py

```diff
@@ -226,6 +307,46 @@
 
 # TODO: Change → api/users/<next_id>
 @USERS_BP.put("api/users/update")
+@doc.summary("Update the details associated with a user.")
+@doc.description(
+    "Update the details associated with a user. Restricted to NEXT standalone mode. Restricted to administrator use."
+)
+@doc.consumes(
+    doc.JsonBody({"next_id": str, "name": str, "username": str, "email": str}),
+    location="body",
+    content_type="application/json",
+    required=True,
+)
+@doc.produces(
+    {"message": "User information was successfully updated."},
+    content_type="application/json",
+    description="A user update status message.",
+)
+@doc.response(
+    400, {"message": str, "code": int}, description="You are not a NEXT Administrator."
+)
+@doc.response(
+    401,
+    {"message": str, "code": int},
+    description="Forbidden: When user unsuccessfully authenticates into NEXT",
+)
+@doc.response(
```

Comment:
we should probably have multiple instances of the response if multiple messages can be generated. If we want to have a single 400 response we should probably refactor the endpoints to return a single generic 400 instead of multiple distinct messages, right?

Reply:
Sounds fine by me, I just combined all scenarios a 400 is thrown into one @doc.response(). I'll change my PR to have a 400 for each unique scenario it is seen being thrown.